src/CMakeLists.txt

```diff
@@ -1,8 +1,8 @@
-if( WIN32 )
-	list(APPEND GAME_SOURCES
-		"${CMAKE_CURRENT_SOURCE_DIR}/wingameicon.o"
-	)
-endif()
+# if( WIN32 )
+# 	list(APPEND GAME_SOURCES
+# 		"${CMAKE_CURRENT_SOURCE_DIR}/wingameicon.o"
+# 	)
+# endif()
```

Comment:
Issues with this...need to fix and uncomment before merge

- [ ] fix game and editor icon